### DISCUSSION TITLE
Is there anyway I can make a page that contains multiple single blog posts like "Showcase" or "About" tab?

### DISCUSSION THREAD

**Discussion by donghhan** [shuding/nextra#2041] [Q&A]
For instance, I would like to make a page that looks like image below.

Which means, that creating a master page that contains multiple number of single blog posts. However, I have no idea how to make master page for that. Can anybody share the idea? 

<img width="1135" alt="image" src="https://github.com/shuding/nextra/assets/84304078/f21ecc34-0590-47a3-a77f-10edfed0ec72">

**Comment by stvnbash** [comment]
Hi, did you figure this out?  This, or even creating a dashboard with dynamic markdown content from multiple pages would be very helpful

**Reply by donghhan** [reply]
No, not yet. I am still figuring out how to do this. Maybe I think I need to customize some configuration by myself. 
I just postponed updating my dev blog posts.
